include/crocoddyl/core/optctrl/shooting.hxx

```diff
@@ -470,6 +482,21 @@ void ShootingProblemTpl<Scalar>::set_terminalModel(boost::shared_ptr<ActionModel
   terminal_data_ = terminal_model_->createData();
 }
 
+template <typename Scalar>
```

Comment:
Thanks for this addition.
Please expose `set_nthreads`,`get_nthreads` and the associated bindings only when `CROCODDYL_WITH_MULTITHREADING` is defined.

Reply:
> Please expose `set_nthreads`,`get_nthreads` and the associated bindings only when `CROCODDYL_WITH_MULTITHREADING` is defined.

I though about this option but I did convince myself that it is not good approach.
For instance, a Python code that uses `nthreads` will be broken if the user doesn't compile crocoddyl with multi-threading support. To avoid that, I still generate the bindings but prints a warning message that tells user *this is useless*.

@proyan  -- after this explanation, do you still think it is better to generate the bindings only when has been defined `CROCODDYL_WITH_MULTITHREADING`?

what is the point of view of others (e.g., @jcarpent @nim65s @wxmerkt )?

Reply:
My approach would have been the same as what you included in this PR to avoid breaking Python code but making it clear that it won't change behaviour.